script/Blast/upgrade/01_configureTokenBridgeFacetBlast.s.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.19;
+
+import { ITokenProxy } from "../../../contracts/diamonds/Token/ITokenProxy.sol";
+import { TokenBridge } from "../../../contracts/facets/Token/TokenBridge.sol";
+import { ITokenBridge } from "../../../contracts/facets/Token/ITokenBridge.sol";
+import { IAxelarExecutable } from "@axelar/interfaces/IAxelarExecutable.sol";
+import { IDiamondWritableInternal } from "@solidstate/contracts/proxy/diamond/writable/IDiamondWritableInternal.sol";
+import { Script, console } from "forge-std/Script.sol";
+import { IToken } from "../../../contracts/facets/Token/IToken.sol";
+import { IDiamondWritable } from "@solidstate/contracts/proxy/diamond/writable/IDiamondWritable.sol";
+
+/// @title ConfigureTokenBridgeFacet
+/// @dev deploys the TokenBridge facet and cuts it onto the TokenProxy diamond
+contract ConfigureTokenBridgeFacet is Script {
+    function run() public {
+        uint256 deployerPrivateKey = vm.envUint("DEPLOYER_KEY");
+        address gateway = vm.envAddress("GATEWAYWAY_ADDRESS");
+        address gasService = vm.envAddress("GAS_SERVICE_ADDRESS");
+        address payable tokenProxyAddress = payable(
+            vm.envAddress("TOKEN_PROXY_ADDRESS")
+        );
+        address multiSigWalletAddress = vm.envAddress(
```

Comment:
This is no long used anywhere.

Reply:
fixed here(caff255801eae8633e349f9a1e048f780be8872d)